developer/src/kmc-ldml/package.json

```diff
@@ -38,10 +38,12 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^20.4.1",
     "@types/semver": "^7.3.12",
+    "abnf": "^4.3.1",
     "c8": "^7.12.0",
     "chalk": "^2.4.2",
     "common-tags": "^1.8.2",
     "mocha": "^8.4.0",
+    "peggy": "^4.2.0",
```

Comment:
These two new deps look okay, although abnf is not widely used

Reply:
it's used in cldr (and deps on peggy). responsive author. i'm a contributor there i think.